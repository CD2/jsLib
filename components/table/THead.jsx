import React from 'react'
import PropTypes from 'prop-types'

export class THead extends React.Component {
  
  static propTypes = {
    children: PropTypes.node,
  }

  render() {
    const { children } = this.props
    return (
      <thead>
        <tr>
          {children}
        </tr>
      </thead>
    )
  }

}
export default THead