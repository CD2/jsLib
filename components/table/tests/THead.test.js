/* eslint-disable react/jsx-filename-extension */
import React from 'react'
import { THead } from '../THead'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`<THead />`, () => {
  const view = shallow((
    <THead />
  ))

  it(`snapshot`, ()=>expect(toJson(view)).toMatchSnapshot())
})