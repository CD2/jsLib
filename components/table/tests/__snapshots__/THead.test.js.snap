// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<THead /> snapshot 1`] = `
<thead>
  <tr />
</thead>
`;
